src/dtx/dtx_rpc.c

```diff
@@ -869,7 +877,9 @@ dtx_refresh_internal(struct ds_cont_child *cont, int *check_count,
 		ABT_rwlock_wrlock(pool->sp_lock);
```

Comment:
It's not from this patch, but I'm wondering why a write lock is required here? When the batched commit/refresh ULTs are running concurrently on VOS xstreams, I think the lock contention will hurt performance badly.

Reply:
Ah, it should be rd lock. I will refresh the patch to fix that.